pkg/services/opsgenie/opsgenie_test.go

```diff
@@ -0,0 +1,134 @@
+package opsgenie
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/containrrr/shoutrrr/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	mockAPIKey = "eb243592-faa2-4ba2-a551q-1afdf565c889"
+)
+
+func TestOpsGenie(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Shoutrrr OpsGenie Suite")
+}
+
+var _ = Describe("the OpsGenie plugin", func() {
```

Comment:
```suggestion
var _ = Describe("the OpsGenie service", func() {
```
Nitpick: we try to consistently use the phrase service